common/environment/setup/install.sh

```diff
@@ -34,17 +34,22 @@ _vsv() {
 	vmkdir etc/sv
 	vcopy "${FILESDIR}/$service" etc/sv
 	if [ ! -L $svdir/run ]; then
+		grep -q 'exec 2>&1' $svdir/run || msg_warn "$pkgver: vsv: service '$service' does not contain 'exec 2>&1' to log stderr\n"
```

Comment:
`&` is a special character, but I am not sure if this matters
```suggestion
		grep -F -q 'exec 2>&1' $svdir/run || msg_warn "$pkgver: vsv: service '$service' does not contain 'exec 2>&1' to log stderr\n"
```